terraform/eks_arm64_cluster_setup/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "eks_cluster_name" {
+  default = "integ-test-arm64-cluster"
+}
+
+variable "region" {
+  default = "us-east-2"
+}
```

Comment:
Is there a reason this is deployed in `us-east-2` instead of `us-west-2` with our other testing clusters?

Reply:
Deployed in `us-east-2` due to limitations of number of vpcs available